packages/components/src/hooks/use-hover.ts

```diff
@@ -40,10 +40,10 @@ export const useHoverCallback = () => {
 
     const handleMouseOver = React.useCallback(() => setValue(true), []);
     const handleMouseOut = React.useCallback(() => setValue(false), []);
-    const ref = React.useRef();
+    const ref: React.MutableRefObject<HTMLElement | null> = React.useRef(null);
```

Comment:
```suggestion
    const ref = React.useRef<HTMLElement | null>(null);
```
@likhith-deriv nit suggestion since it's working in the same way, just a bit shorter🙂